packages/vscode-dependencies-validation/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
we use the root launch.json
maybe remove this?
also add any none root `.vscode` to .gitignore?

Reply:
Removed.
What do you mean by "none root .vscode" ?